GiniVision.podspec

```
@@ -29,13 +29,13 @@ The Gini Vision Library for iOS provides functionality to capture documents with
   s.subspec 'Networking' do |networking|
     networking.source_files = 'GiniVision/Classes/Networking/*.swift', 'GiniVision/Classes/Networking/Extensions/*.swift'
     networking.dependency "GiniVision/Core"
-    networking.dependency 'Gini/DocumentsAPI', '~> 0.5.3'
+    networking.dependency 'Gini/DocumentsAPI', '>= 0.5.3'
```

I'd prefer using the [optimistic operator](https://guides.cocoapods.org/using/the-podfile.html#specifying-pod-versions) ([here's a nice vide from Google ](https://www.youtube.com/watch?v=x4ARXyovvPc) about it):
```suggestion
    networking.dependency 'Gini/DocumentsAPI', '~> 1.0.0'
```

This makes sure it will update only to versions 1.0.x, so it will use patches but not update the minor or major version numbers.

Using `>= 0.5.3` will update also to 2.0.0 or 3.0.0 which will likely break stuff.

---

yep, let's stay on the safe side